hyperjumptech/monika
# fix: monika parses integer to string

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Fix Monika parses integer in the request body to a string. This will resolves #922.

## How did you implement / how did you fix it  
1.  Create a function that generates chaining request body
2. Test the function

## How to test  
1. Create a http server that receives request with body that consists of property with integer data type
2. Check if Monika send the request with the correct data type

### Monika Config Sample
![Screenshot from 2022-10-21 18-31-34](https://user-images.githubusercontent.com/15191978/197188095-af000845-f323-498f-9f45-01a27eb3d178.png)

### Before
![Screenshot from 2022-10-21 18-31-34](https://user-images.githubusercontent.com/15191978/197188171-65985c17-5648-4688-aed7-7497d7b2fac5.png)

### After
![Screenshot from 2022-10-21 18-32-39](https://user-images.githubusercontent.com/15191978/197188183-61ec39d6-b9ab-4a33-b8c7-b886077ac0b8.png)
